development-docs/systemtests/labels/label.md

```diff
@@ -0,0 +1,23 @@
+# Label
+
+#### Label: `label`
```

Comment:
I think this might be grouped with `annotation` label?

Reply:
>  think this might be grouped with annotation label?

Not sure, I think sometimes in tests you use only Labels (f.e., in this test `testLabelsExistenceAndManipulation`) we verify only labels. On the other hand, there are a few tests, where only annotation is used... :)